http-lobby/src/main/java/org/triplea/server/error/report/upload/ErrorReport.java

```diff
@@ -0,0 +1,31 @@
+package org.triplea.server.error.report.upload;
```

Comment:
Chipping away at this I think is still appropriate. To another extent keeping things localized and grouping when it is time can be a good route too.

With that said, I think I'm more convinced to call the sub-project `http-server` instead of `http-lobby`. Not a very good assumption it will always be a lobby component (might do dice rolls for example). At the same time I'm not super jazzed to call the sub-project specific implementation/technology details. I don't think we'll have more than one http server, so likely `http-server` is a good enough name, I'm a bit tempted to call it `http-server-app` to emphasize it is the full app, but that seems to be getting wordy. 

I also looked at the package structure, I suspect we may want more kinds of reports and those should go under a common package structure, so I also swapped `error.report` for `reporting.error`

To give a sense of where this is heading too, the additional http server implementation will be dropping in, and a client and common data repo will also be landing:

![screenshot from 2018-09-02 21-18-21](https://user-images.githubusercontent.com/12397753/44967549-fa571900-aef6-11e8-9245-3c9c2a9e1f7c.png)

![screenshot from 2018-09-02 21-14-32](https://user-images.githubusercontent.com/12397753/44967550-fa571900-aef6-11e8-8e66-6783e0cad21d.png)